### PR TITLE
EIG-2501

### PR DESCRIPTION
**Pull request by matthewbennett**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by smenzer** [comment]
please remove `package-lock.json` from the PR, thanks!

**Review by harpere — CHANGES_REQUESTED** [review]
please remove `package-lock.json`

**Comment by matthewbennett** [comment]
package-lock.json has been restored

File: test/spec/modules/spotxBidAdapter_spec.js [hunk 1]
```diff
@@ -564,6 +565,7 @@ describe('the spotx adapter', function () {
       expect(scriptTag.getAttribute('data-spotx_content_width')).to.equal('400');
       expect(scriptTag.getAttribute('data-spotx_content_height')).to.equal('300');
       expect(scriptTag.getAttribute('data-spotx_ad_mute')).to.equal('1');
+      expect(scriptTag.getAttribute('data-spotx_custom_container_styles')).to.equal('postion: absolute;');
```
Comment (harpere) [inline test/spec/modules/spotxBidAdapter_spec.js:568]: the first `_` here is a `-` above.  Looks like this is causing tests to fail.

**Comment by harpere** [comment]
@matthewbennett - ping - test needs fixing

File: test/spec/modules/spotxBidAdapter_spec.js (hunk 1 above)
Comment (matthewbennett) [reply test/spec/modules/spotxBidAdapter_spec.js:568]: I would like to abandon this pull request.

**Comment by matthewbennett** [comment]
I would like to abandon this pull request.

**Pull request closed without merging**